src/activities/nine_men_morris/NineMenMorris.qml

```diff
@@ -304,81 +307,16 @@ ActivityBase {
                 right: background.right
                 rightMargin: 5
             }
-            radius: 5
             state: "second"
```

Comment:
I don't think the state is still needed, it is handled in the ScoreItem, there should be no more references in the activity